render.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 
 	"github.com/onsi/ginkgo/types"
+	"bufio"
```

Comment:
Could you move this import to `line 4` to match this structure:

```go
import (
	"bufio"
	"fmt"
	"strings"

	"github.com/onsi/ginkgo/types"
)
```

Thank you.